oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/log/AbstractLogRecord.java

```diff
@@ -44,6 +44,7 @@
     public static final String SPAN_ID = "span_id";
     public static final String CONTENT_TYPE = "content_type";
     public static final String CONTENT = "content";
+    public static final String CONTENT_TYPE_CLASS = "content_type_class";
```

Comment:
I think we don't need this? You run `#toJSON` already.

Reply:
Yes, forgot to remove, now removed